src/screens/UserPortal/UserScreen/UserScreen.tsx

```diff
@@ -131,7 +131,7 @@ const UserScreen = (): JSX.Element => {
       >
         <div className="d-flex justify-content-between align-items-center">
           <div style={{ flex: 1 }}>
-            <h1>{titleKey !== 'home' ? t('title') : ''}</h1>
+            <h1>{titleKey !== 'home' ? t('title') : t('title')}</h1>
```

Comment:
The condition gives same result for both the cases so please replace with 
```suggestion
            <h1>{t('title')}</h1>
```

Reply:
Yes, I did it. 
![image](https://github.com/user-attachments/assets/13792d19-66d3-4f5a-ba4f-d828fba3c46a)